planning/backlog.md

```diff
@@ -13,7 +13,18 @@
 
 ## Must-Haves
 
+- As a **user** I want to a logo so that I can identify the website easily.
```

Comment:
without "to" or "to see" please @yildiztugba

Reply:
aa yes, i ll change